docs/videos/healthcare-and-life-sciences/Factor-Xa-example/README.md

```diff
@@ -0,0 +1,43 @@
+# Factor Xa Example
+
+This example demonstrates a real life example of simulating the Factor Xa
```

Comment:
nit: this example demonstrates a real life example...
what about: this example demonstrates a real life case of ... ?

Reply:
Done